components/brave_wallet_ui/options/panel-titles.ts

```diff
@@ -0,0 +1,25 @@
+import { PanelTitleObjectType } from '../constants/types'
+import { getLocale } from '../../common/locale'
+
+export const PanelTitles = (): PanelTitleObjectType[] => [
```

Comment:
Assuming we really need this as a function for lazy evaluation, let's memoize wherever this is called in components.

Reply:
As discussed via slack, for a timing issue related to `getLocale` this has to be a function. Not able to memoize this in the reducer where it's being called.